script/1_0_2current_en_script/Common/00_02_hidden.txt

```diff
@@ -34,16 +34,16 @@ And in my place, their figure is torn to shreds.
 だから 空を見あげることにした.
 So I raised my gaze to the heavens.
 ---<宙|ソラ>を<塞|ふさ>ぐ<天蓋|てんがい>
-―――a canopy that seals off the sky
+―――a canopy sealing off the sky.
```

Comment:
I think I prefer the previous here. Also, there's no period in the JP, so I would say don't add one here.